lib/controllers/reviews-controller.js

```diff
@@ -96,9 +97,8 @@ export class BareReviewsController extends React.Component {
     if (initThreadID && initThreadID !== prevProps.initThreadID) {
       this.setState(prev => {
         prev.threadIDsOpen.add(initThreadID);
+        this.highlightThread(initThreadID);
         return {commentSectionOpen: true, scrollToThreadID: initThreadID};
-      }, () => {
-        setTimeout(() => this.setState({scrollToThreadID: null}), FLASH_DELAY);
```

Comment:
I wish we had a way to collapse the `setState` updater functions from here and `highlightThread` into a single, atomic transaction. I'm pretty sure that's how it's working anyway and this is much cleaner but still 😛